src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/MissingOverrideCheck.java

```diff
@@ -81,6 +107,24 @@
  *       value="true"/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * class Test {
+ *
+ *     &#47;** {&#64;inheritDoc} *&#47;
+ *     public void test() { // violation
+ *
+ *     }
+ * }
+ *
+ * class Test extends SuperClass {
+ *
+ *     &#47;** {&#64;inheritDoc} *&#47;
+ *     public void test() { // OK
```

Comment:
Doc for this property says that
`This property will only check classes, interfaces, etc. that do not contain the extends or implements keyword or are not anonymous classes.`
please add here comment why this is not cause violation

Reply:
I am done.